functions/nvm.fish

```diff
@@ -121,9 +121,9 @@ function _nvm_ls -a query
     ' <$index 2>/dev/null
 end
 
-function _nvm_use
+function _nvm_resolve_version -a requested_ver
     set -l index (_nvm_get_index); or return
-    set -l ver (command awk -v ver="$argv[1]" '
+    set -l ver (command awk -v ver="$requested_ver" '
```

Comment:
@briancavalier Can we use `$argv[1]` here? Just for consistency with the existing code (not because it's a good idea to have nameless arguments).

Reply:
Sure.